src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -104,9 +115,9 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     <button
       className={containerClasses}
       style={roomInlineStyles}
-      onClick={isCovertRoom ? noop : selectRoomWithSound}
-      onMouseEnter={isCovertRoom ? noop : handleRoomHovered}
-      onMouseLeave={isCovertRoom ? noop : handleRoomUnhovered}
+      onClick={shouldBeClickable ? selectRoomWithSound : undefined}
+      onMouseEnter={shouldBeClickable ? handleRoomHovered : undefined}
+      onMouseLeave={shouldBeClickable ? handleRoomUnhovered : undefined}
```

Comment:
[NOTE]
This way works OK.

[OOS]
Alternatively these checks could've been done inside the respective `useCallback()`s as encapsulated logic, but that will also obscure what is now readily visible.